package.json

```diff
@@ -52,7 +52,7 @@
     "@nextcloud/event-bus": "^3.3.1",
     "@nextcloud/files": "^3.10.1",
     "@nextcloud/initial-state": "^2.2.0",
-    "@nextcloud/l10n": "^3.1.0",
+    "@nextcloud/l10n": "^3.2.0",
```

Comment:
adjustment isn't needed for that, as in vue-lib?

Reply:
new vue-lib is needed

Reply:
Drafted in another PR